cypress/e2e/code-of-conduct.cy.js

```diff
@@ -0,0 +1,102 @@
+  
+      it('should verify UI elements on the Code of Conduct page', () => {
+        cy.contains("Mentorship Code of Conduct").should('be.visible');
+        cy.contains("Code Of Conduct for Mentees").should('be.visible');
+        cy.contains("Code Of Conduct for Mentors").should('be.visible');
+        cy.contains('Show More').should('be.visible');
+        cy.contains('Read').should('be.visible');
+      });
+  
+      it('should find Slack link within #mentee-section and verify URL and text', () => {
+        cy.get('#mentee-section')
+        .find('a[href="https://bit.ly/women_coding_community"]')
+        .should('exist')
+        .contains('Slack')
+      })
+  
+    })
+  
+    describe('Test "Show more," "Show less," for mentee and mentor', () => {
+      
+      it('should verify the existence of the anchor tag with id "#btn-mentee-learn-more" and text "Show More"', () => {
+        cy.get('a#btn-mentee-learn-more').should('be.visible').contains('Show More').should('be.visible');
+        cy.get('a#btn-mentee-learn-more').click();    
+        cy.get('a#btn-mentee-show-less:contains("Show Less")', { timeout: 10000 }).should('be.visible');
+      });
+    
+      it('should verify the existence of the anchor tag with id "#btn-mentee-learn-more" and text "Show More"', () => {
+        cy.get('a#btn-mentor-learn-more').should('be.visible').contains('Show More').should('be.visible');
+        cy.get('a#btn-mentor-learn-more').click();
+        cy.get('a#btn-mentee-show-less:contains("Show Less")', { timeout: 10000 }).should('exist');    
+      });
+  
+    })
+  
+    describe('Test "read" functionality', () => {
+      it('should open another page when clicking on the "Read" link', () => {
+        cy.contains('Read').click();  
+        cy.url().should('eq', 'https://womencodingcommunity.com/code-of-conduct');
+      });
+    })
+  
+    describe('Verify all footer UI elements', () => {
+  
+      it('should verify the visibility of footer UI elements', () => {
+        cy.get('.brand').should('be.visible');
+        cy.contains("Women Coding Community is a not-for-profit organisation.").should('be.visible');
+        cy.contains('Follow Us').should('be.visible');
+        cy.contains('Join us on social media and stay tuned.').should('be.visible');
+        cy.contains('© 2024 Women Coding Community').should('be.visible');
+      });
+    
+      it('should verify LinkedIn link', () => {
+        cy.get('.network a[href="https://www.linkedin.com/company/womencodingcommunity"]')
+          .should('be.visible')
+          .should('exist')
+          .should('have.attr', 'target', '_blank')
+      });
+  
+      it('should verify Slack link', () => {
+        cy.get('.network a[href="https://bit.ly/women_coding_community"]')
+          .should('exist')
+          .should('have.attr', 'target', '_blank')
+          .should('have.attr', 'href', 'https://bit.ly/women_coding_community');
```

Comment:
Here for example you use two times slack link, good practice to extract to const, or global const and re-use. Because this will be showing in many pages. It will be easier to maintain in a single place in case it change.

Reply:
I will take care of this in another PR.